packages/app-form-builder/src/admin/components/FormEditor/Draggable.tsx

```diff
@@ -11,18 +10,23 @@ export type DraggableChildrenFunction = (params: {
     drag: ConnectDragSource;
 }) => ReactElement;
 
-interface BeginDragProps {
+export interface BeginDragProps {
     ui?: "row" | "field" | "step";
     pos?: {
         row: number;
         index?: number;
     };
-    formStep?: FbFormStep;
     name?: string;
+    id?: string;
+    // "container" contains info about source element.
+    container?: {
+        type: "step" | "conditionGroup";
```

Comment:
Again a use case where we could use a type that could live in `types.ts` file. Let's do that and also make sure to use the type in `packages/app-form-builder/src/admin/components/FormEditor/Droppable.tsx`.